### ISSUE TITLE
311: PM Team Meeting Agenda and Notes

### ISSUE THREAD

**Issue by EchoProject** [hackforla/311-data#1136]
### Overview
Meeting Agendas for PM/Org meetings

#### To create a new agenda
- [ ] Copy the template below into a new comment
- [ ] Edit the text you pasted with new items for your agenda
- [ ] Add the link to the new comment with its date under Resources/Instructions

### Resources/Instructions
- #1407  
The prework should be done before the meeting so that it can be discussed during the meeting.  It usually takes about 5 min, if your team reaches all issues being labeled and milestones setup.  During cleanup it usually takes about 30 min weekly.

#### Template for next meeting agenda
```
## [Date] Agenda and Minutes

### Prework
- [ ] label audit #1134
- [ ] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column

### Roll Call

#### Recurring items: 
 - [ ] review any issues that are in the columns
   - [ ] [new issue approval](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox
      - [ ] [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Alexandra](https://github.com/hackforla/311-data/issues/assigned/alexulven)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from last meeting

#### New Items

#### FYIs

#### Actions taken or notes about stuff from the meeting:

#### New Task Items
```

### Meetings Past and future
[2021-09-28 ](https://github.com/hackforla/311-data/issues/1136#issuecomment-930370696)
[2021-10-12](https://github.com/hackforla/311-data/issues/1136#issuecomment-941751222)
[2021-10-19](https://github.com/hackforla/311-data/issues/1136#issuecomment-947108630)
[2021-10-26](https://github.com/hackforla/311-data/issues/1136#issuecomment-952466471)
[2021-11-02](https://github.com/hackforla/311-data/issues/1136#issuecomment-958363600)
[2021-11-09](https://github.com/hackforla/311-data/issues/1136#issuecomment-966733747)
[2021-11-16](https://github.com/hackforla/311-data/issues/1136#issuecomment-970733866)
[2021-12-06](https://github.com/hackforla/311-data/issues/1136#issuecomment-991382145)
[2021-12-17](https://github.com/hackforla/311-data/issues/1136#issuecomment-998354092)
[2021-12-20](https://github.com/hackforla/311-data/issues/1136#issuecomment-998354843)
[2021-12-27](https://github.com/hackforla/311-data/issues/1136#issuecomment-998355021)

**Comment by EchoProject** [comment]
2021-09-28

1. Bonnie and Eric reviewed the Shared Google Drive migration issue. Found subfolders missing from the [New Drive](https://drive.google.com/drive/folders/1ObUz4mTQmSCwND_WQcOXAuCeKpCwNzmz) within the Data Analysis and UI/UX folders. Next step: Eric will complete the folder recreation. 
2. Updated [311-Data team meeting schedule](https://github.com/hackforla/VRMS/issues/769) issue and sent a request to VRMS to update the project page. PM team meeting changed to Tuesdays at 11:00 AM, All team meeting unchanged to Thursdays at 6:30 PM, and deleted Data Science team meeting. Next step: Share with the team. 
3. Jason (VRMS engineer) gave a demo on his last day. David (VRMS engineer) will be back from vacation soon. Next step: None. 
4. Quick look at Elizabeth's [Menu contrast issue](https://github.com/hackforla/311-data/issues/1126). Question for UI on request type change to checkboxes. Next step: Review during the weekly team meeting. 
5.  Bike and Pedestrian data request issue has been sent to the Data Science team board and closed for 311-Data. Next steps: None. 
6. Bonnie created a new issue for District Types. This is outside of the 311-Data scope right now. Sent to John(?)... Next step: None. 
7. Neighborhood Council relationship management needed. Typically handled by PMs. Next step: Eric will gather contact information (names, addresses, phone numbers, emails, etc.), create CRM in Google Sheet to track last communications. 
8. Discussed long-term planning to utilize shapefile to allow stakeholders to view 311-Data by different district types and compare different district types. Next step: Eric will evaluate each issue on the Project Board to identify issues outside the scope of the 311-Data team and relabel Move to Data Science for Bonnie to review. This will create space allowing the team to focus on the v2 launch and prioritize our roadmap for the next 3 - 6 months.

**Comment by EchoProject** [comment]
2021-10-12 

1. The automated feature of "Stale" posts was requested to be removed from the Project Board. 
2. Bonnie and Eric reviewed "In Progress" Issues on the Project Board and commented for an update on Issues past 7 days without any update. 
3. Bonnie closed the "Document 311-Data Meeting Times ..." Issue. 
4. Bonnie updated the Project Board automation setting to move Issues to the "Done" column once Closed. 
5. Bonnie unassigned Issues in the "Prioritized Backlog" column.
6. Reviewed Issues in the "Done" column that was still Open. Moved Issues with incomplete comments to the "Review" column. 
7. Created new Issues for requests made by UI/UX https://github.com/hackforla/311-data/issues/1139 , https://github.com/hackforla/311-data/issues/1140 , https://github.com/hackforla/311-data/issues/1141 and moved to the "Review" column for additional feedback from Design.

**Comment by EchoProject** [comment]
2021-11-02 

1. Quick run-through of the Review column on the Project Board. Made a note to discuss with the team the PR review process. Commented on a FrontEnd issue for additional discussion details in the comments. 
2. Reviewed line items from https://github.com/hackforla/311-data/issues/801#issue-692620071 mainly regarding some UX notes from v1. 
3. Based on one of the line items from the above: Eric will create an issue to have "How to Submit a Service Request" added to the FAQ. 
4. After some design review, suggest a change to the about page using an Analysis Page color palette.  
5. Eric will reach out to Dana Chen to schedule to re-discover intelligence from prior user testing and research. 
6. Eric will reach out to Adam Kendis to confirm scheduling in regards to new developer onboarding. 
7. Reviewed Google Drive Migration video tutorial and File Owner Sheet. 
8. Eric will make some edits to the video for another review before reaching out to File Owners.

**Comment by EchoProject** [comment]
2021-11-09 

1. Discussed having a suggestion box for new features that are not derived directly from research. We can possibly create a new label that states if the Issue is research-based. We can create a spreadsheet that maps ideas and research. 
2. The Peer Review documentation will be added to the Wiki once the Wiki is ready to be worked on. At this time, we are waiting for the completion of the Google Drive migration so that we can own and work with the existing files. 
3. Discovered existing Figma account for 311-Data. We will be migrating all previous designs to this new account. Bonnie will work with Dana and Nate on the migration. Eric will invite current active team members to the new account at the team meeting. 
4. Icebox clean up.

**Comment by EchoProject** [comment]
2021-11-16

1. PM Team meeting will be changed to Tuesdays at 3 pm. 
2. 311-Data Team meeting Zoom link has been updated and the host key is available in the 311-Data 1Password vault. 
3. Bonnie set up a meeting request with Dana to discuss Google Drive and Figma migrations. 
4. Updated the #808 to include further details on dependency, dev tickets, and testing. 
5. Moved #1154 and #1155 to the Review Board from Icebox to confirm status.

**Comment by EchoProject** [comment]
2021-12-6
1. Changed PM Team meeting scheduled for Dec. and Jan. 12/17 will be on Friday at 4pm. 12/20 and 12/27 will be on Monday at 3pm. Starting in January, the meetings will remain on Fridays at 4pm. 
2. Reviewed label audit issue #1137 and added another set of links for auditing. 
3. Discovered [311 MVP ](https://github.com/hackforla/311-data/projects/4) project board that has been closed. Went through open issues. These issues will need to be reviewed with Adam and moved to the current board as necessary.

**Comment by ExperimentsInHonesty** [comment]
## Agenda and Minutes 2021-12-17
(this was original posted on 12-17 on another issue https://github.com/hackforla/311-data/issues/1161#issuecomment-997103445

### Prework
- [ ] label audit #1134

### Roll Call
Bonnie, Lexi, Eric

#### Recurring items: 
 - [x] review any issues that are in the [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [x] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [x] [Alexandra](https://github.com/hackforla/311-data/issues/assigned/alexulven)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### New Items
- [ ] Icebox review to see what is going on with the issues with no dependency
   - pushed to next meeting

#### Actions taken or notes about stuff from the meeting:

- We performed the audit during the meeting.  There was no significant change since last week.
- Brought back a [job posting](https://github.com/hackforla/UI-UX/projects/3#card-56238941) from the filled column and edited it.  To recruit for designer

#### New Task Items
- [ ] Add to next weeks agenda, scheduling next steps for driving the labels missing count down, if it has not changed.
- [ ] Lexie to Follow up with Chelsey to see all the issues she is currently assigned to that affect the project and prioritize what we need to know. https://github.com/hackforla/311-data/issues/1137
    - [ ] Has she worked on this at all (including just writing documentation https://github.com/hackforla/311-data/issues/699) and can she estimate size
- [ ] Eric will review the icebox for the issues that don't have a dependency to see what we need to do next https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency 
- [ ] Make roadmap issue and add links
    - [Open Community Survey Roadmap](https://docs.google.com/spreadsheets/d/163RPNWPhkeah2OvGCqjByMukYduXk66XY7ZpcpLCzXU/edit#gid=617281993)
   - [Guides Team Roadmap Example](https://docs.google.com/spreadsheets/d/1xYUlIKsfE2-CeExIQ8i3oLD8pC7hjMTPM2YLSMFA8_s/edit#gid=0)
   - Guides Team Milestones https://github.com/hackforla/guides/milestones
- [ ] Eric to combo these two issue together (this agenda issue and issue #1136)

**Comment by ExperimentsInHonesty** [comment]
## Agenda and Minutes 2021-12-20
(this was original posted on 12-20 on another issue [#1161 (comment)](https://github.com/hackforla/311-data/issues/1161#issuecomment-998336109)

### Prework
- [x] label audit #1134

### Roll Call
Bonnie, Eric

#### Recurring items: 
 - [x] review any issues that are in the [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
 - [x] Accountability and Support Check
    - [x] Review assignments for each PM
       - [x] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [x] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [x] [Alexandra](https://github.com/hackforla/311-data/issues/assigned/alexulven)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### New Items
- [x] Icebox review to see what is going on with the [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)

#### Actions taken or notes about stuff from the meeting:
- Lexie has not yet left any notes on her issue to follow up with Chelsey.  Bonnie asked for update.  Issue will be carried into next meeting
- Answered Erics questions about how to combo the agenda issues and meeting notes

#### New Task Items
- [x] scheduling next steps for driving the labels missing count down, if it has not changed.
- [x] Lets see if Lexie to Follow up with Chelsey to see all the issues she is currently assigned to that affect the project and prioritize what we need to know. https://github.com/hackforla/311-data/issues/1137
    - [ ] Has she worked on this at all (including just writing documentation https://github.com/hackforla/311-data/issues/699) and can she estimate size
- [x] Eric will review the icebox for the issues that don't have a dependency to see what we need to do next https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency 
- [x] Did a roadmap issue get made - Yes it did https://github.com/hackforla/311-data/issues/1162
- [x] Did Eric to combo these two issue together (this agenda issue and issue #1136)

**Comment by ExperimentsInHonesty** [comment]
## Agenda and Minutes 

### Prework
- [ ] label audit #1134
- [ ] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column

### Roll Call


#### Recurring items: 
 - [ ] review any issues that are in the 
   - [ ] [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Alexandra](https://github.com/hackforla/311-data/issues/assigned/alexulven)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from last meeting
- [ ] see if Lexie to Follow up with Chelsey to see all the issues she is currently assigned to that affect the project and prioritize what we need to know. https://github.com/hackforla/311-data/issues/1137
- [ ] scheduling next steps for driving the labels missing count down, if it has not changed.
- [ ] Did Eric to combo these two issue together (this agenda issue and issue #1136) 

#### New Items

#### Actions taken or notes about stuff from the meeting:

#### New Task Items

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-01-10

### Prework
- [x] label audit #1134
- [] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column
  - [] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
Eric
Bonnie
Lexie


#### Recurring items: 
 - [] review any issues that are in the 
   - [] [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [] Icebox review
      - [] [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from last meeting
- [x] see if Lexie to Follow up with Chelsey to see all the issues she is currently assigned to that affect the project and prioritize what we need to know. https://github.com/hackforla/311-data/issues/1137
- [ ] scheduling next steps for driving the labels missing count down, if it has not changed.


#### New Items

#### Actions taken or notes about stuff from the meeting:
Numbers from label audit went down by one issue, we did not spend any time reducing the numbers over the last 4 weeks.

#### Notes From Meeting
-  Decided to bring in new dev to focus on Plotly to decide if we will continue using the tech

#### New Task Items

- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 

### Next agenda
https://github.com/hackforla/311-data/issues/1137#issuecomment-1007108249

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-01-10

### Prework
- [ ] label audit #1134
- [ ] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
Eric
Bonnie
Lexie


#### Recurring items: 
 - [ ] review any issues that are in the 
   - [ ] [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Lexie]()
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from last meeting
- [ ] scheduling next steps for driving the labels missing count down, if it has not changed.


#### New Items

#### Actions taken or notes about stuff from the meeting:
Numbers from label audit went down by three issues and increased by 1, we did not spend any time reducing the numbers over the last 4 weeks.

#### Notes From Meeting
-  Decided to bring in new dev to focus on Plotly to decide if we will continue using the tech
- Bonnie contacted remaining persons on the Drive Migration contacts list
- Eric is taking over Wiki update 

#### New Task Items

- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 

- [ ] Eric will create an issue for the v1 site not rendering for some users
- [ ] Eric will create a Sheet that lists out Wiki page titles and a description 

### Next agenda

**Comment by ExperimentsInHonesty** [comment]
FYI for next meeting
I updated this card, its still had the old PMs name on it. https://github.com/hackforla/engineering/projects/2#card-54125928, 

Item for meeting
we should look over all the open roles posts we have and make sure they are up to date

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-01-21

### Prework
- [ ] label audit #1134
- [ ] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call



#### Recurring items: 
 - [ ] review any issues that are in the 
   - [ ] [new issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Lexie](https://github.com/hackforla/311-data/issues/assigned/alexulven)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] finish adding labels to the issue that are still missing them (see audit links above)
- [ ] review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from the last meeting
- [ ] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
- [ ] Be sure to talk about #1137. Tabling due to time, but need to address Plotly Dash.
- [ ] We should look over all the open roles posts we have and make sure they are up to date
- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 


#### New Items


#### Notes From Meeting


#### New Task Items


### Next agenda

**Comment by ExperimentsInHonesty** [comment]
next meeting 
https://github.com/hackforla/access-the-data/issues/34#issuecomment-1027350641

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-02-04

### Prework
- [ ] label audit #1134
- [ ] Review the icebox to make sure that if dependencies have been met, the issue is moved to new issue approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
Eric 


#### Recurring items: 
 - [ ] Review any issues that are in the 
   - [ ] [New issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [Issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Lexie](https://github.com/hackforla/311-data/issues/assigned/alexulven)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] Finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items carried over from the last meeting
- [ ] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
- [ ] Be sure to talk about #1137. Tabling due to time, but need to address Plotly Dash.
- [ ] We should look over all the open roles posts we have and make sure they are up to date
- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 
- [ ] Work through issues from v1 launch project board to add to post v2 roadmap 


#### New Items

- [ ] From Bonnie's comment above: 
       Q: Use cases of the 311 data visualization tool
       A: I have added this to the 311-data PM meeting agenda, we will answer it here


#### Notes From Meeting


#### New Task Items


#### Next agenda

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-02-11

### Prework:
- [x] label audit #1134
- [x] Review the Icebox; if dependencies have been met, then issue is moved to the New Issue Approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
- [x] Bonnie
- [x] Eric
- [ ] Lexie
- [ ] Pras


#### Recurring items: 
 - [ ] Review any issues that are in the 
   - [ ] [New issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [Issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Lexie](https://github.com/hackforla/311-data/issues/assigned/alexulven)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] Finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items Carried Over From the Last Meeting:
- [ ] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
- [ ] Be sure to talk about #1137. Tabling due to time, but need to address Plotly Dash.
- [ ] We should look over all the open roles posts we have and make sure they are up to date
- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 
- [ ] Work through issues from v1 launch project board to add to post v2 roadmap 
- [ ] Clean and queue up development tickets for new engineers joining the team 
   - [ ] Update Milestones for issues in [Prioritized Backlog](https://github.com/hackforla/311data/projects/10card_filter_query=milestone%3A%22z.v2+launch%22) 
- [ ] Create issues for missing features in the [v1 vs. v2 Feature Comparison](https://docs.google.com/spreadsheets/d/1KWPIoo0skY4I_Xi7bFTPbyjQ6QiyVq4RML_t3IViAqs/edit#gid=0) 
- [ ] Finalize/ publish [Meet Up event](https://www.meetup.com/hackforla/events/drafts/)
- [ ] Create slide presentation for Meet Up event 



#### Items to Discuss at This Meeting: 
- [ ] From Bonnie's comment above: 
       Q: Use cases of the 311 data visualization tool
       A: I have added this to the 311-data PM meeting agenda, we will answer it here


#### Notes From This Meeting: 



#### New Agenda/ Task Items for Next Meeting:

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-02-18

### Prework:
- [ ] label audit #1134
- [ ] Review the Icebox; if dependencies have been met, then issue is moved to the New Issue Approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
- [x] Bonnie
- [x] Eric


#### Recurring items: 
 - [ ] Review any issues that are in the 
   - [ ] [New issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [Issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Lexie](https://github.com/hackforla/311-data/issues/assigned/alexulven)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] Finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items Carried Over From the Last Meeting (Try to keep this list empty):
- [ ] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
   - [ ] Have a model now, go through history, 311 interactions with NCs

v2 Data visualizations 
- [ ] Make multiple issues for devs around Plotlydash configurations
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 
- [ ] Work through issues from v1 launch project board to add to post v2 roadmap 
- [ ] Be sure to talk about #1137. Tabling due to time, but need to address Plotly Dash.

Dev Recruiting
- [ ] Clean and queue up development tickets for new engineers joining the team 
   - [ ] Update Milestones for issues in [Prioritized Backlog](https://github.com/hackforla/311data/projects/10card_filter_query=milestone%3A%22z.v2+launch%22) 
- [ ] We should look over all the open roles posts we have and make sure they are up to date
- [ ] Finalize/ publish [Meet Up event](https://www.meetup.com/hackforla/events/drafts/)
- [ ] Create slide presentation for Meet Up event 


#### Items to Discuss at This Meeting: 
- [ ] [From Bonnie's comment above](https://github.com/hackforla/access-the-data/issues/34#issuecomment-1027350641): 
       Q: Use cases of the 311 data visualization tool
       A: I have added this to the 311-data PM meeting agenda, we will answer it here 
           - Dependent on Eric completing the digging up on the original research. 
- [ ] Create issues for missing features in the [v1 vs. v2 Feature Comparison](https://docs.google.com/spreadsheets/d/1KWPIoo0skY4I_Xi7bFTPbyjQ6QiyVq4RML_t3IViAqs/edit#gid=0) 


#### Notes From This Meeting: 



#### New Agenda/ Task Items for Next Meeting:

**Comment by ExperimentsInHonesty** [comment]
Please add to our next agenda
Let's discuss you adding this [CTA](https://docs.google.com/presentation/d/1UDHwTQ1C3y01CiNS0mvPjNBHJFlm2Txs/preview) to your team agenda

**Comment by EchoProject** [comment]
## Agenda and Minutes 2021-02-18

### Prework:
- [ ] label audit #1134
- [ ] Review the Icebox; if dependencies have been met, then the issue is moved to the New Issue Approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
- [x] Bonnie
- [x] Eric


#### Recurring items: 
 - [ ] Review any issues that are in the 
   - [ ] [New issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [Issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [x] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
   - [x] Have a model now, go through history, 311 interactions with NCs
   - [ ] Progress update
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] Finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items Carried Over From the Last Meeting (Try to keep this list empty):

v2 Data visualizations 
- [ ] Make multiple issues for devs around Plotlydash configurations; need to know which issues require a data scientist and which need an engineer. Some issues are [posted here](https://github.com/hackforla/data-science/projects/2) for DS team to self-assign.
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 

Dev Recruiting:
- [ ] Eric will create issues for recruiting for roles 
- [ ] Finalize/ publish [Meet Up event](https://www.meetup.com/hackforla/events/drafts/)
- [ ] Create slide presentation for Meet Up event 


#### Items to Discuss at This Meeting: 
- [ ] Wiki directory topology 


#### Notes From This Meeting: 



#### New Agenda/ Task Items for Next Meeting:

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-03-25

### Prework:
- [ ] label audit #1134
- [ ] Review the Icebox; if dependencies have been met, then the issue is moved to the New Issue Approval column
  - [ ] [All issues that are not marked as being post V2](https://github.com/hackforla/311-data/projects/10?card_filter_query=-milestone%3A%22post+v2%22)

### Roll Call
- [ ] Bonnie
- [x] Eric


#### Recurring items: 
 - [ ] Review any issues that are in the 
   - [ ] [New issue approval column](https://github.com/hackforla/311-data/projects/10#column-15704782)
   - [ ] Icebox review
      - [ ] [Issues with no dependency](https://github.com/hackforla/311-data/projects/10?card_filter_query=-label%3Adependency#column-11518033)
 - [x] Bonnie working with Bernard from Open Community Survey. He is adding all the NC logos and we need to add to the figma the names of the NCS we have had relationships with and link a document with details to the figma #571 
   - [x] Have a model now, go through history, 311 interactions with NCs
   - [ ] Progress update
 - [ ] Accountability and Support Check
    - [ ] Review assignments for each PM
       - [ ] [Bonnie](https://github.com/hackforla/311-data/issues/assigned/ExperimentsInHonesty)
       - [ ] [Eric](https://github.com/hackforla/311-data/issues/assigned/EchoProject)
       - [ ] [Pras](https://github.com/hackforla/311-data/issues/assigned/pras4u)
- [ ] Finish adding labels to the issue that are still missing them (see audit links above)
- [ ] Review issues with [epic label](https://github.com/hackforla/311-data/issues?q=is%3Aopen+is%3Aissue+label%3AEpic) and define how to break down further

#### Items Carried Over From the Last Meeting (Try to keep this list empty):

v2 Data visualizations 
- [ ] Make multiple issues for devs around Plotlydash configurations; need to know which issues require a data scientist and which need an engineer. Some issues are [posted here](https://github.com/hackforla/data-science/projects/2) for DS team to self-assign.
    - [ ] Legends on the charts SCSS
    - [ ] SCSS to make it look the same as the rest of the product 
    - [ ] List of features 

Dev Recruiting:
- [ ] Eric will create issues for recruiting for roles 
- [ ] Finalize/ publish [Meet Up event](https://www.meetup.com/hackforla/events/drafts/)
- [ ] Create slide presentation for Meet Up event 


#### Items to Discuss at This Meeting: 
- [ ] Wiki directory topology 


#### Notes From This Meeting: 



#### New Agenda/ Task Items for Next Meeting:

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-04-18

#### Items to Discuss at This Meeting: 
**Problems that need to be fixed:** 
  1. We are at zero velocity with no clear objective 
  2. We don't spend meeting time productively (we do things rather than make decisions; update non-essentials) 
  3. Emphasis on prioritizing backlog when we don't have a defined product vision 
  4. 4 months with no Dev Team 

**Product Strategy Plan** 
  - [ ] Product meeting structure - prioritization - Highest Level needs first 
  - [ ] Define (write down) product (all decisions must map back to these statements): 
     - [ ] Mission: We provide data visualization and analytics tools for neighborhood councilpersons to help them make better decisions for their constituents through local initiatives.   
     - [ ] Vision: To provide analytics tools for local governments across the United States. 
     - [ ] Northstar Metric: Number of query Exports/ Downloads 

**Product Development Plan** 
  - [ ] Focus on MVP launch ASAP (i.e. < 3 months) rather than mapping features from v1 for v2 
     - [ ] Add features through iteration based on user feedback NOT existing Issue ideas 
       - [ ] Existing Issue MAY be added into sprint cycles ONLY if they meet requirements 
  - [ ] Monthly sprints - Eric will create Issue and supporting documents
     - [ ] Week 1 - Product Planning - Team Leads (key features and fixes to include in next cycle)  
     - [ ] Week 2 - All teams Stand Up 
     - [ ] Week 3 - Peer Review and Testing   
     - [ ] Week 4 - Merge & Deploy next sprint (new Issues assigned from prior month planning)
        - [ ] 1-5 Issues MAX per Month per Member 
        - [ ] 1 Peer Review & Merge per Month (Eng Activity)
     - [ ] Week 5 - Address additional bugs & tech debt 
    
**Project Management Plan**
  - [ ] Organize new Google Drive to optimize document finding - Eric is working on Issue
  - [ ] Update Wiki to optimize onboarding - Eric is working on Issue
  - [ ] Project Board prioritization 
     - [ ] Make prioritization DECISIONS during meeting times, DON'T work on tickets 
     - [ ] Melissa and Anupriya will be updating Issues to meet standards for their Roles (exercise in writing)  
     - [ ] We need to select the MOST important Issues to "fix" that align with the product goal 

**Recruitment Plan**        
 - [ ] Recruit Researcher & Front/Back/Fullstack (Lead) developers - Eric is working on Issue
    - [ ] Post bulletin via Slack channels to find existing HFLA Fullstack Eng that wants leadership opportunity 
    - [ ] Schedule Tutorial with Matt Webster to record an onboarding session with new TL 
    - [ ] Push new volunteers into 311-data team's onboarding pipeline

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-03-25 

#### Updates:
1. Josh is working on a first draft of the Reports Dashboard on BI Tool because of the load on his system to run Plotly and the complexity of writing the code. Need to create an issue for this so we can track progress and have history. 
2. May run into scaling issues using pandas library given dataset size. We'll need to revisit this with Nich before v2 launch. 
3. Nich fixed the build pipeline and merged Edwin's last [Hover Contrast PR](https://github.com/hackforla/311-data/pull/1148). 
4. Eric created an Issue for [backfilling missing data from the pipeline](https://github.com/hackforla/311-data/issues/1190).
5. Eric informed Nich of the [v1 contact us form bug](https://github.com/hackforla/311-data/issues/1099). 
6. Eric will be meeting with Melissa next week to discuss design onboarding to prepare materials for updating the Wiki and consolidating Google Drive. We'll also be reorganizing the Figma to read more clearly. 

#### Needs to get done today: 
1. First Thursday product planning meeting confirmation 
2. Determine priority Issues (5-10) for eng team to launch v2 MVP (phase 1 of the v2 roadmap)  
3. If time permits: 
    - Define product vision in writing (i.e. Democratize service request data and improve community data literacy) 
    - Define North Star metric (i.e. Number of exported Reports) 

#### Needs to get done this month: 
1. Schedule tutorial/ Q&A with Matt Webster (or other prior dev) to fill knowledge gaps on eng team. Target date: Late May, early June. 
2. Eric will work with Melissa to start harmonizing the design of informational pages on v2 for launch. 
5. Eric will prepare a user-research plan; needs contact info, would be nice to have a researcher 
6. Eric will create v2 roadmap via Miro - wants to remove old contributors

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-05-20 

#### Updates: 
1. Eric met with Design Team on Figma housekeeping and [onboarding improvements](https://github.com/hackforla/311-data/issues/1201)
2. New Issue for the frontend to create https://github.com/hackforla/311-data/issues/1202 
3. New Issue for the frontend to create https://github.com/hackforla/311-data/issues/1200 

#### Needs to get done today: 
- [ ] 1. District shapefiles - Several existing issues 
            a. https://github.com/hackforla/311-data/issues/849
            b. https://github.com/hackforla/311-data/issues/941
            c. https://github.com/hackforla/data-science/issues/141
            d. https://github.com/hackforla/311-data/issues/1203 
- [ ] 2. Determine priority Issues (5-10) for eng team to launch v2 MVP (phase 1 of the v2 roadmap) see below 
- [ ] 3. Define product vision in writing (i.e. Democratize service request data and improve community data literacy) 
- [ ] 4. Define North Star metric (i.e. Number of exported Reports) 

#### Needs to get done this month: 
- [ ] 1. Schedule tutorial/ Q&A with Matt Webster to fill knowledge gaps on eng team. Target date: Mid June. 
- [ ] 5. Eric will prepare a user-research plan; needs contact info and would be nice to have a UX researcher 
- [ ] 6. Eric will create a v2 roadmap via Miro - wants to remove old contributors
- [ ] 8. Recruiting pipeline - [Democracy Lab Hackathon](https://www.democracylab.org/events)

**Comment by ExperimentsInHonesty** [comment]
Please open a [new issue on hackforla website repo](https://github.com/hackforla/website/issues/new?assignees=&labels=P-Feature%3A+Project+Info+and+Page%2C+role%3A+back+end%2FdevOps%2C+role%3A+front+end%2C+Size%3A+Small%2C+size%3A+0.5pt%2C+time+sensitive&template=project-profile-card-review-and-update.md&title=Update+Project+Profile%3A+%5BProject+Name%5D) to update the hack for la website with the team members

**Comment by ExperimentsInHonesty** [comment]
We need to ask Nich to create a doc like this
https://github.com/hackforla/tdm-calculator/wiki/Software-Architecture-and-Development
And also include a list of all the technologies used.

**Comment by EchoProject** [comment]
@ExperimentsInHonesty 
@nichhk and I were just talking about something similar in that the team should spend time, moving forward, improving the code quality of our codebase. 

As of now, the soft plan is to have devs work on documentation of certain parts of the system while they complete tickets. I agree that having a single source of truth document for our architecture and system design is valuable and important. 

It may take some time as our devs ramp up. For now, we can start a Wiki document and have Nich make the contributions as the team makes progress. 

Does this sound like a good enough plan? I'm open to suggestions.

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-06-03 

#### Updates: 
1. 

#### Needs to get done today: 
- [ ] Issue transfer from DS to 311 

#### Needs to get done sooner than later: 
- [ ] Review and revise, if necessary, DS issue related to Plotly 
- [ ] Schedule tutorial/ Q&A with Matt Webster to fill knowledge gaps for the Eng team. Target date: Late June - Waiting for response 
- [ ] Eric will prepare a user-research plan; needs contact info and would be nice to have a UX researcher - Not started
- [ ] Create Plotly tutorials

**Comment by ExperimentsInHonesty** [comment]
https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-06-23 

#### Updates: 
1. v2 menu will have two surfaces depending on whether Address or District Type tab is selected 
2. Prod site is still down. Eng is waiting on Matt Webster to send SSH keys and config files 
3. New data scientist - Susan Kolim 

#### Needs to get done: 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 
- [ ] Review Issues transferred from DS for completeness, then move to PB 
- [ ] Reschedule 1-1 with Bonnie to 2nd week of July

#### Needs to get done sooner than later: 
Vision & Strategy: 
- [ ] OKR - [H2 2022 Objective Key Results]
- [ ] Roadmap - [H2 2022 Roadmap](https://docs.google.com/spreadsheets/d/1H5KMz4pgKCalLSnPIHqDqkW0_viVzLnt3OgYklpoadk/edit#gid=0) 
Engineering & Data Science: 
- [ ] Standardize CSS code for dev site components for Figma Design System 
- [ ] Standardize CSS code for Plotly components for Figma Design System
Project Organization:  
- [ ] Update project Wiki and Readme to reflect current status 
- [ ] Create Onboarding Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
- [ ] Create Recruiting Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
Board Management: 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 
- [ ] 
Research: 
- [ ] Prepare a user-research plan 
- [ ] Get NC contact list info  
- [ ] Recruit a UX researcher 
Miscellaneous 
- [ ] Edit and post video from knowledge transfer session with Matt Webster

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-06-23 

#### Updates: 
1. v2 menu 
2. Prod site is still down. Eng is waiting on Matt Webster to send SSH keys and config files 
3. New data scientist - Susan Kolim 

#### Needs to get done: 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 
- [ ] SSH Key and config files from Matt Webster - Bonnie 

#### Needs to get done sooner than later: 
Vision & Strategy: 
- [ ] OKR - [H2 2022 Objective Key Results]
- [ ] Roadmap - [H2 2022 Roadmap](https://docs.google.com/spreadsheets/d/1H5KMz4pgKCalLSnPIHqDqkW0_viVzLnt3OgYklpoadk/edit#gid=0) 
Engineering & Data Science: 
- [ ] Standardize CSS code for dev site components for Figma Design System 
- [ ] Standardize CSS code for Plotly components for Figma Design System
Project Organization:  
- [ ] Update project Wiki and Readme to reflect current status 
- [ ] Create Onboarding Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
- [ ] Create Recruiting Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
Board Management: 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 
- [ ] 
Research: 
- [ ] Prepare a user-research plan 
- [ ] Get NC contact list info  
- [ ] Recruit a UX researcher 
Miscellaneous 
- [ ] Edit and post video from knowledge transfer session with Matt Webster

**Comment by EchoProject** [comment]
## Agenda and Minutes 2022-07-21 

#### Updates: 
Desktop and mobile menu designs are being created. 

6. We can begin alpha testing (dogfooding) the website before beta testing with real users. (Lucky Parking DS contact)
7. Product offering - DS deep dive: Our DS team will conduct an analysis for partnered NCs (and other districts) and provide a report (via blog) with the findings. The blog reports allow others to run similar diagnostics and community initiatives. Estimated for 2 to 3 months out. (Bad Example: Already tried this. Gave the NC report and didn't receive feedback on when they used the analysis. Good Example: Clean Streets Report Analysis - where to put trash cans - still open)  

#### Needs to get done: 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 
- [x] SSH Key and config files from Matt Webster - Bonnie - Moving past v1 and focusing efforts on v2 
- [ ] Get data contact at the city of LA - Bonnie will write to them - Send a paragraph about what we need 
- [ ] Remove old members from invites. 

#### Needs to get done sooner than later: 
Vision & Strategy: 
- [ ] OKR - [H2 2022 Objective Key Results]
- [ ] Roadmap - [H2 2022 Roadmap] - Meeting with Nich next week to finalize priorities and project estimations  (https://docs.google.com/spreadsheets/d/1H5KMz4pgKCalLSnPIHqDqkW0_viVzLnt3OgYklpoadk/edit#gid=0) 

Engineering & Data Science: 
- [ ] Standardize CSS code for dev site components for Figma Design System 
- [ ] Standardize CSS code for Plotly components for Figma Design System
- [ ] DS API for GitHub labels 

Design: 
- [ ] Menu re-design: Desktop AND Mobile   
- [ ] Holistic coloring - Dark Mode -> Light Mode - Pages look mostly done, need to send to eng. 
- [ ] Standardize Figma Design System to include CSS code 

Project Organization:  
- [ ] **Update project Wiki and Readme to reflect the current status** 
- [ ] Create Onboarding Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
- [ ] Create Recruiting Issues for Product, Design, Research, Frontend, Fullstack, and Data Science 
- [ ] https://github.com/hackforla/311-data/issues/1134#issuecomment-1159201206 

Research: 
- [ ] Prepare a user-research plan 
- [ ] Get NC contact list info  
- [ ] Recruit a UX researcher 

Miscellaneous 
- [ ] Edit and post video from knowledge transfer session with Matt Webster (mostly a DevOps convo)

**Comment by ExperimentsInHonesty** [comment]
I changed the label you had `Feature: Meeting Notes/Agendas` to `Feature: Agenda` to confirm to all the other projects, so that we can auto display them on the website.

**Comment by ryanfchase** [comment]
For some reason this was closed as "Not Planned" (we mass closed many tickets). I will get this linked back to the most recent agenda
